app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -123,6 +124,38 @@ func (s *ForwardingRuleSyncer) GetLoadBalancerStatus(lbName string) (*status.Loa
 	return status, nil
 }
 
+// ListForwardingRules gets all global forwarding rules and returns info about ones we created.
+func (s *ForwardingRuleSyncer) ListForwardingRules() (string, error) {
+	var rules *compute.ForwardingRuleList
+	var err error
+	if rules, err = s.frp.ListGlobalForwardingRules(); err != nil {
```

Comment:
Depending on which PR goes in first, we will need to update this to handle HTTPS ingresses.
HTTPS ingresses create 2 forwarding rules: one for HTTP and one for HTTPS.